assets/js/confirm_submit.js

```diff
@@ -0,0 +1,12 @@
+const formToConfirm = document.getElementById("js-confirm-submit");
+
+const confirmSubmit = function(){
+  if(confirm("Do you really want to submit the pick?"))
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)